### ISSUE TITLE
Typing support

### ISSUE THREAD

**Issue by Sigmanificient** [GiorgosXou/TUIFIManager#43]
I'd like to see (maybe add by myself) typing support as it is a good practice and would help to understand a bit more of what is going on, often allowing to spot more complex issues and code simplifying.

What are your thoughts about it?

**Comment by GiorgosXou** [comment]
Yeah typing seems like a good thing although i don't think that it is a priority for now... 

Also, if you have multiple warnings in your editor because of typing issues and you use pyright you can disable `typeChekingMode` by setting it to `off`
